docs/ref-info/ref-server-side-token-generation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Server-Side Token Generation
+description: Information for publishers about generating the token on the server side.
+hide_table_of_contents: false
+sidebar_position: 06
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Link from '@docusaurus/Link';
+
+# Server-Side Token Generation
+
+If you're a publisher setting up a client-server or server-side UID2 integration, the first step is to generate the UID2 token on your server. Then, you can keep the token refreshed either on the client side or server side, and pass the token to the client side for sending to the RTB bidstream.
+
+There are two approaches for publishers generating UID2 tokens on the server side by providing directly identifying information (<Link href="../ref-info/glossary-uid#gl-dii">DII</Link>) (email address or phone number):
+
+- Integration with an SDK
+- Direct integration to API endpoints
+
+:::warning
+For security reasons, the API key and secret used in token generation *must* be called on the server side. Do not store these values on the client side, whether on a web page, mobile app, or in Prebid. For details, see [Security of API Key and Client Secret](../getting-started/gs-credentials.md#security-of-api-key-and-client-secret).
+:::
+
+Options are summarized in the following table.
+
+| Integration Solution  | Generate Token | Refresh Token |
+| :--- | :--- |  :--- |
+| [UID2 SDK for Java](../sdks/uid2-sdk-ref-java.md) | ✅ | ✅ |
+| [UID2 SDK for Python](../sdks/uid2-sdk-ref-python.md) | ✅ | ✅ |
+| [Direct integration (API endpoints with custom code)](../endpoints/post-token-generate.md) | ✅ | ✅ |
+
+Whatever integration option you choose to generate the <Link href="../ref-info/glossary-uid#gl-identity">identity</Link> (UID2 token and associated values), you'll need to implement one of the following:
+
+- Call the [POST&nbsp;/token/generate](../endpoints/post-token-generate.md) endpoint.
+
+  The identity output that you need for the rest of this guide is the content inside the body section of a successful endpoint response. For an example, see [Successful Response](../endpoints/post-token-generate.md#successful-response).
+
+-  Use one of the Publisher Client classes, in one of the UID2 server-side SDKs. These classes simplify the request into a single method call. 
```

Comment:
Can we move this above `call \token\generate`? Easier for participants to use the SDKs rather than write their own encryption/decryption code.

Reply:
Great idea. Done, thx.